src/js/lib/pages/github/issue.js

```diff
@@ -8,6 +9,120 @@ import K2pickerType from '../../../module/K2pickertype/K2pickertype';
 import ToggleReview from '../../../module/ToggleReview/ToggleReview';
 import K2comments from '../../../module/K2comments/K2comments';
 import ONYXKEYS from '../../../ONYXKEYS';
+import * as API from '../../api';
+
+let clearErrorTimeoutID;
+function catchError(e) {
+    $('.gh-header-actions .k2-element').remove();
+    $('.gh-header-actions').append('<span class="alert k2-element">OOPS!</span>');
+    console.error(e);
+    clearTimeout(clearErrorTimeoutID);
+    clearErrorTimeoutID = setTimeout(() => {
+        $('.gh-header-actions .k2-element').remove();
+    }, 30000);
+}
+
+/**
+ * Sets the owner of an issue when it doesn't have an owner yet
+ * @param {String} owner to set
+ */
+function setOwner(owner) {
+    API.getCurrentIssueDescription()
+        .then((response) => {
+            const ghDescription = response.data.body;
+            const newDescription = `${ghDescription}
+
+Current Issue Owner: @${owner}`;
```

Comment:
I asked about that in the Slack thread and no one seemed to interested in it. I don't mind putting it in one though if you think it would be helpful.

Reply:
Oh I should have specified 🤦‍♂️ I think the main value in using a details block is that it's collapsable if there's extra text telling the user not to edit the value. Just putting it inside the details block with no other info doesn't seem that useful. I realize though that this suggestion is solving a problem we don't really have, so that's my bad.

Reply:
Oh got it, I missed this comment before merging. I actually think it's fine if someone chooses to manually edit it to change the person. It could be that not everyone is using the k2 extension.